packages/@aws-cdk/aws-cognito/lib/user-pool-idps/google.ts

```diff
@@ -15,8 +16,15 @@ export interface UserPoolIdentityProviderGoogleProps extends UserPoolIdentityPro
   /**
    * The client secret to be accompanied with clientId for Google APIs to authenticate the client.
    * @see https://developers.google.com/identity/sign-in/web/sign-in
+   * @default none
```

Comment:
```suggestion
   * @default none
   * @deprecated use clientSecretValue instead
```
I think we should just deprecate this since you can use `clientSecretValue: SecretValue.unsafePlainText()`

Reply:
Done